src/Containers/VendorsList/VendorsHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, ButtonProps } from '../../Inputs/Button';
+import { Heading } from '../../Text/Heading';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { flex } from '../../Utils/Mixins';
+
+export interface VendorsHeaderProps
+    extends MainInterface,
+        ResponsiveInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    headerText: string;
+    rightButtonText: string;
+    leftButtonProps: ButtonProps;
```

Comment:
Btw I like to preface all my interfaces with I to make it eaiser to auto correct